src/common/TextInputComp/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Controller } from 'react-hook-form';
+import { TextInput } from 'hds-react';
+
+type propTypes = {
+  name: string;
+  id: string;
+  control: any;
+  rules?: any;
+  defaultValue: string;
+  label: string;
+  invalid?: boolean;
+  errorMsg?: string;
+};
+const TextInputComp: React.FC<propTypes> = (props) => {
```

Comment:
Please rename to TextInput. And dev.hel.fi coding standard is to not use index.tsx file: 
https://dev.hel.fi/coding-standards#naming